Library/Homebrew/shims/scm/git

```diff
@@ -9,6 +9,12 @@ quiet_safe_cd() {
   cd "$1" >/dev/null || { echo "Error: failed to cd to $1" >&2; exit 1; }
 }
 
+abspath() {
+  local dir="$(quiet_safe_cd "${1%/*}/" && pwd -P)"
```

Comment:
That's not what I suggested, though. The whole point of my suggestion was to construct an absolute path *without* resolving any symbolic links and the `pwd -P` in here at least partially does exactly that.

It may not resolve `/usr/local/bin/git`, but I'm pretty sure it will resolve `/usr/local/opt/git/bin/git` to `/usr/local/Cellar/git/<version>/bin/git` which seems undesirable.